gen/calc.py

```diff
@@ -421,6 +421,64 @@ def calculate_cluster_docker_registry_enabled(cluster_docker_registry_url):
     return 'false' if cluster_docker_registry_url == '' else 'true'
 
 
+def validate_cosmos_config(cosmos_config):
```

Comment:
Same story for calculate functions, btw. https://github.com/dcos/dcos/blob/master/gen/internals.py#L61